content/assets/blog/phoenix-case-study/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Phoenix: An end-to-end AI powered Social Media analysis platform powering peacebuilding
+efforts"
+date: "2025-02-10T10:16:57Z"
+description: "Phoenix is an open-source AI platform developed by Build Up and datavaluepeople to
+aid peacebuilding through social media analysis. Launched in January 2025, it addresses challenges
+like large data processing, limited technical expertise, and under-resourced-language
+classification. Key features include a user-friendly interface, secure authentication, data
+scraping, and multilingual classification. Phoenix has already supported hundreds peacebuilders,
```

Comment:
Throughout sometimes "multilingual" is used, sometimes "multi-language" is used. Probs better to pick one and stick with it.

Reply:
93b3bbf